src/issue.js

```diff
@@ -23,7 +23,12 @@ class Issue {
         const isCore = isCommitter(this.issue.author_association, this.issue.user.login);
 
         if (!isCore) {
-            // avoid opening an issue with no template through `Reference in new issue`
+            // check if the title is valid
+            if (this.isMissingTitle()) {
+                this.addLabels.push(label.MISSING_TITLE);
+                return;
+            }
+            // prevent from opening an issue with no template via `Reference in new issue` button
             if (!this.isUsingTemplate()) {
```

Comment:
I think we can disable template keyword based template checking temporary to avoid bot closing the issue(and forget why) unexpectedly when we change the template in the future. But we can still pay attention if there are invalid issue that not follow the template